### PR TITLE
(fix) [SD-3387] Fixing for Ingest Provider Creation failure. 

### PR DESCRIPTION
**Pull request by sivakuna-aap**
Reason: Registration of Feeding Services and Feed Parsers happen later after registering the Ingest Provider Resource. Thus causing the value of 'allowed' property to be empty.

### CODE REVIEW

File: superdesk/io/ingest_provider_model.py [hunk 1]
```diff
@@ -43,13 +43,11 @@ def __init__(self, endpoint_name, app, service, endpoint_schema=None):
             'source': required_string,
             'feeding_service': {
                 'type': 'string',
-                'required': True,
-                'allowed': list(allowed_feeding_services)
```
Comment (mugurrus) [inline superdesk/io/ingest_provider_model.py]: Why allow invalid values?

Comment (sivakuna-aap) [inline superdesk/io/ingest_provider_model.py]: Handling invalid values in `on_create` and `on_update` instead of Cerberus.

Comment (mugurrus) [inline superdesk/io/ingest_provider_model.py]: Sorry, I saw later that you do the validation in on_ methods

**Comment by mugurrus** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**